addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -41,16 +41,16 @@
         <div class="container">
             <div class="row s_nb_column_fixed">
                 <div class="o_masonry_col o_snippet_not_selectable col-lg-4">
-                    <img class="img img-fluid d-block rounded" src="/web/image/website.library_image_03" data-index="0" data-name="Image" alt=""/>
-                    <img class="img img-fluid d-block rounded" src="/web/image/website.library_image_10" data-index="3" data-name="Image" alt=""/>
+                    <img class="img img-fluid d-block rounded o_image_popup" src="/web/image/website.library_image_03" data-index="0" data-name="Image" alt=""/>
+                    <img class="img img-fluid d-block rounded o_image_popup" src="/web/image/website.library_image_10" data-index="3" data-name="Image" alt=""/>
                 </div>
                 <div class="o_masonry_col o_snippet_not_selectable col-lg-4">
-                    <img class="img img-fluid d-block rounded" src="/web/image/website.library_image_13" data-index="1" data-name="Image" alt=""/>
-                    <img class="img img-fluid d-block rounded" src="/web/image/website.library_image_05" data-index="4" data-name="Image" alt=""/>
+                    <img class="img img-fluid d-block rounded o_image_popup" src="/web/image/website.library_image_13" data-index="1" data-name="Image" alt=""/>
+                    <img class="img img-fluid d-block rounded o_image_popup" src="/web/image/website.library_image_05" data-index="4" data-name="Image" alt=""/>
                 </div>
                 <div class="o_masonry_col o_snippet_not_selectable col-lg-4">
-                    <img class="img img-fluid d-block rounded" src="/web/image/website.library_image_14" data-index="2" data-name="Image" alt=""/>
-                    <img class="img img-fluid d-block rounded" src="/web/image/website.library_image_16" data-index="5" data-name="Image" alt=""/>
+                    <img class="img img-fluid d-block rounded o_image_popup" src="/web/image/website.library_image_14" data-index="2" data-name="Image" alt=""/>
+                    <img class="img img-fluid d-block rounded o_image_popup" src="/web/image/website.library_image_16" data-index="5" data-name="Image" alt=""/>
```

Comment:
Suppose a website has been created by drag dropping this snippet, how the user will be informed whether this snippet is outdated or not ? Please use `data-vml` with specific version when changing a snippet.